ExporterStreamProcessorTest is unstable

`#shouldConfigureAllExportersProperlyOnStart` fails sometimes with the stacktrace below. It looks as if there is a race condition between the stream processor being recovered and the test asserting that the controller has been injected.

```
[ERROR] Tests run: 18, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 3.47 s <<< FAILURE! - in io.zeebe.broker.exporter.stream.ExporterStreamProcessorTest
[ERROR] shouldConfigureAllExportersProperlyOnStart(io.zeebe.broker.exporter.stream.ExporterStreamProcessorTest)  Time elapsed: 0.088 s  <<< FAILURE!
java.lang.AssertionError:

Expecting actual not to be null
        at io.zeebe.broker.exporter.stream.ExporterStreamProcessorTest.shouldConfigureAllExportersProperlyOnStart(ExporterStreamProcessorTest.java:140)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
        at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
        at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
        at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
        at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
        at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
        at org.junit.rules.RunRules.evaluate(RunRules.java:20)
        at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
        at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
        at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
        at org.junit.runners.Suite.runChild(Suite.java:128)
        at org.junit.runners.Suite.runChild(Suite.java:27)
        at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
        at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
        at org.apache.maven.surefire.junitcore.JUnitCore.run(JUnitCore.java:55)
        at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.createRequestAndRun(JUnitCoreWrapper.java:137)
        at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.executeEager(JUnitCoreWrapper.java:107)
        at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:83)
        at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:75)
        at org.apache.maven.surefire.junitcore.JUnitCoreProvider.invoke(JUnitCoreProvider.java:157)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)

```

--- a/broker-core/src/test/java/io/zeebe/broker/exporter/stream/ExporterStreamProcessorTest.java
+++ b/broker-core/src/test/java/io/zeebe/broker/exporter/stream/ExporterStreamProcessorTest.java
@@ -119,16 +119,22 @@ public class ExporterStreamProcessorTest {
   private List<ControlledTestExporter> exporters;
 
   @Test
-  public void shouldConfigureAllExportersProperlyOnStart() {
+  public void shouldConfigureAllExportersProperlyOnStart() throws InterruptedException {
     // given
     final Map[] arguments = new Map[] {newConfig("foo", "bar"), newConfig("bar", "foo")};
     final List<ExporterDescriptor> descriptors = createMockedExporters(arguments);
     final ExporterStreamProcessor processor =
         new ExporterStreamProcessor(PARTITION_ID, descriptors);
 
+    final CountDownLatch latch = new CountDownLatch(exporters.size());
+    for (ControlledTestExporter exporter : exporters) {
+      exporter.onOpen(c -> latch.countDown());
+    }
+
     // when
     final StreamProcessorControl control = rule.initStreamProcessor(e -> processor);
     control.start();
+    assertThat(latch.await(5, TimeUnit.SECONDS)).isTrue();
 
     // then
     for (int i = 0; i < exporters.size(); i++) {

